modules/bbp-simulation/src/test/resources/data/hbphackaton/simulation/morphologyrelease/v0.1.0/all-fields.json

```diff
@@ -15,7 +15,17 @@
   "@id": "{{base}}/data/hbphackathon/simulation/morphologyrelease/v0.1.0/12e68e2d-47fe-47c1-bc5f-195d914d00ae",
   "name": "Morphology release SSCx O1.v6a",
   "distribution": {
-    "accessURL": "file:///projectpath/entities/morphologies/2017.10.31"
+    "accessURL": "file:///projectpath/entities/morphologies/2017.10.31",
+    "mediaType": [
```

Comment:
Are you sure validation is working here? Because I had problems submitting data with "application/swc" media type. I had to submit the whole string, like: "application/swc,application/neurolucida,application/h5,application/neuroml"

Reply:
I corrected the schemas. one allowed media type should validate now.